app/views/layouts/application.html.erb

```diff
@@ -63,5 +63,17 @@ window.current_page_name = "<%= params[:controller] %>/<%= params[:action] %>";
 
 <%= render "layouts/footer" %>
 
+<% if @queue_stats %>
```

Comment:
What is this, and how is it different from the /status page?  (That page, I think, holds more information about whose jobs are being graded...)